### PR TITLE
Updated ABIS and addresses for the pool upgrade

### PR DESCRIPTION
**Pull request by roxdanila**

### CODE REVIEW

File: version-data/data.json [hunk 1]
```diff
@@ -9,18 +9,18 @@
...
       },
       {
-        "address": "0x58676340f458b36997608672be6548c92ce50714",
+        "address": "0xcafea63ad9028fa022bf07eed9adb4dccc6c9bbc",
         "code": "CL",
         "smartContractCode": "CL",
         "contractName": "claim1",
```
Comment (horodnicdragos) [inline version-data/data.json:15]: We should see where this property is used and try to update it to the same name that we use in smart-contracts.

**Review by horodnicdragos — CHANGES_REQUESTED** [review]

File: version-data/data.json [hunk 2]
```diff
@@ -30,11 +30,25 @@
...
+      },
+      {
+        "address": "0xcafea6A946406b0B48a77348A4F70DFEeD0f4664",
+        "code": "SWAP",
+        "smartContractCode": "SWAP",
+        "contractName": "SwapAgent",
+        "contractAbi": "[{\"constant\":true,\"inputs\":[],\"name\":\"MAX_LIQUIDITY_RATIO\",\"outputs\":[{\"internalType\":\"uint256\",\"name\":\"\",\"type\":\"uint256\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"},{\"constant\":true,\"inputs\":[{\"internalType\":\"uint256\",\"name\":\"tokenAmountIn\",\"type\":\"uint256\"},{\"internalType\":\"contract IERC20\",\"name\":\"fromToken\",\"type\":\"IERC20\"},{\"internalType\":\"contract IERC20\",\"name\":\"toToken\",\"type\":\"IERC20\"}],\"name\":\"getSwapQuote\",\"outputs\":[{\"internalType\":\"uint256\",\"name\":\"tokenAmountOut\",\"type\":\"uint256\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"},{\"constant\":true,\"inputs\":[],\"name\":\"router\",\"outputs\":[{\"internalType\":\"contract IUniswapV2Router02\",\"name\":\"\",\"type\":\"IUniswapV2Router02\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"}]"
       },
```
Comment (shark0der) [inline version-data/data.json:52]: SwapAgent cannot be used from off-chain. Please remove.

File: version-data/data.json [hunk 3]
```diff
@@ -72,18 +86,18 @@
...
..."internalType\":\"address\",\"name\":\"masterAddress\",\"type\":\"address\"},{\"internalType\":\"address\",\"name\":\"_dai\",\"type\":\"address\"}],\"payable\":false,\"stateMutability\":\"nonpayable\",\"type\":\"constructor\"},{\"constant\":false,\"inputs\":[],\"name\":\"changeDependentContractAddress\",\"outputs\":[],\"payable\":false,\"stateMutability\":\"nonpayable\",\"type\":\"function\"},{\"constant\":false,\"inputs\":[{\"internalType\":\"address\",\"name\":\"masterAddress\",\"type\":\"address\"}],\"name\":\"changeMasterAddress\",\"outputs\":[],\"payable\":false,\"stateMutability\":\"nonpayable\",\"type\":\"function\"},{\"constant\":true,\"inputs\":[],\"name\":\"dai\",\"outputs\":[{\"internalType\":\"contract IERC20\",\"name\":\"\",\"type\":\"address\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"},{\"constant\":true,\"inputs\":[],\"name\":\"master\",\"outputs\":[{\"internalType\":\"contract INXMMaster\",\"name\":\"\",\"type\":\"address\"}],\"payable\":false,\"stateMutability\":\"view\",\"type\":\"function\"},{\"constant\":false,\"inputs\":[],\"name\":\"sendEther\",\"outputs\":[],\"payable\":true,\"stateMutability\":\"payable\",\"type\":\"function\"}]"
```
Comment (shark0der) [inline version-data/data.json:100]: Malformed JSON: missing `"contractAbi": `

Comment (roxdanila) [reply version-data/data.json:100]: uhm, check the line above

File: version-data/data.json (hunk 1 above)
Comment (roxdanila) [reply version-data/data.json]: This isn't used anywhere, but I updated the names everywhere for clarity

**Pull request merged**